### PR TITLE
Display OAuth2 endpoints

### PR DESCRIPTION
**Pull request by dominikglandorf**
In tool management show urls of oauth2 endpoints.

Needs backend PR: https://github.com/schul-cloud/schulcloud-server/pull/749

### CODE REVIEW

File: views/tools/tools.hbs [hunk 1]
```diff
@@ -10,6 +10,19 @@
...
     {{#content "page"}}
         <div class="container-fluid">
+            <div class="row mobile-no-margin-right">
+                <div class="card h-100">
+                    <div class="card-block">
+                        <h5 class="card-title">OAuth2 Endpunkte</h5>
+                        <div class="card-text">
+                            <h6>Authorization: <a href="{{baseUrl}}/oauth2/auth">{{baseUrl}}/oauth2/auth</a></h6>
+                            <h6>Token: <a href="{{baseUrl}}/oauth2/token/">{{baseUrl}}/oauth2/token</a></h6>
+                            <h6>Userinfo: <a href="{{baseUrl}}/oauth2/token">{{baseUrl}}/userinfo</a></h6>
```
Comment (malcm) [inline views/tools/tools.hbs]: href is wrong here

File: controllers/tools.js [hunk 2]
```diff
@@ -225,14 +229,17 @@ router.all('/', function (req, res, next) {
     const itemsPerPage = (req.query.limit || 10);
```
Comment (malcm) [inline controllers/tools.js]: Big code duplication with /search ... create a function. Changes need to be made only once

File: controllers/tools.js [hunk 3]
```diff
@@ -179,18 +179,21 @@ router.get('/search' , function (req, res, next) {
     const itemsPerPage = 10;
```
Comment (malcm) [inline controllers/tools.js]: If I am on the search route an create a new tool it leads to an error (Not found) ... don't use current URL location when POST the modal for creation of a new tool.

File: views/tools/tools.hbs (hunk 1 above)
Comment (dominikglandorf) [reply views/tools/tools.hbs]: is correct when configured in server config (services.hydra)

**Pull request merged**